project.clj

```diff
@@ -1,40 +1,45 @@
-(defn- with-ns [n] (str "org.broadinstitute.firecloud-ui." n))
-
 (defproject org.broadinstitute/firecloud-ui "0.0.1"
   :dependencies
   [
-   [dmohs/react "0.2.11"]
-   [org.clojure/clojure "1.7.0"]
-   [org.clojure/clojurescript "1.7.228"]
+   [dmohs/react "1.0.2+15.0.2"]
+   [org.clojure/clojure "1.8.0"]
+   [org.clojure/clojurescript "1.9.293"]
```

Comment:
we're now way ahead of the web repl at http://clojurescript.net/ ... know of any others?